azext_iot/central/_help.py

```diff
@@ -276,30 +278,31 @@ def _load_central_api_token_help():
         "iot central api-token"
     ] = """
         type: group
-        short-summary: Create and Manage API tokens.
+        short-summary: Manage API tokens for your IoT Central application.
+        long-summary: IoT Central allows you to generate and manage API tokens to be used to access the IoT Central API. More information about APIs can be found at https://aka.ms/iotcentraldocsapi.
     """
 
     helps[
         "iot central api-token create"
     ] = """
         type: command
-        short-summary: Create a new API token in the application
-        long-summary: The only time you will see the value of this token is when creating the token. Ensure you store this token somewhere securely, as if you lose it, you will need to create another.
+        short-summary: Generate an API token associated with your IoT Central application.
+        long-summary: |
+            Note: Write down your token once it's been generated as you won't be able to retrieve it again.
```

Comment:
nit - this Note is pretty important. I'd argue we might want to put it in Short Summary.

At the least we don't need the `: |`, thats usually for super long multi-line comments

Reply:
Fixed.